cmd/validate/image.go

```diff
@@ -45,6 +46,7 @@ import (
 type imageValidationFunc func(context.Context, app.SnapshotComponent, *app.SnapshotSpec, policy.Policy, []evaluator.Evaluator, bool) (*output.Output, error)
 
 var newConftestEvaluator = evaluator.NewConftestEvaluator
+var newOPAEvaluator = evaluator.NewOPAEvaluator
```

Comment:
Would it be more efficient to not instantiate both of them?

Reply:
Oh, never mind. I see that we're not calling `evaluator.NewConftestEvaluator`, we're just referencing it here. Disregard the above.

Reply:
yeah, I think using the vars made it easier to test.